doc/index.rst

```diff
@@ -6,38 +6,65 @@
 lsst.ts.ATDome
 ##############
 
-Controller for the LSST auxiliary telescope dome.
+.. image:: https://img.shields.io/badge/SAL\ Interface-green.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/ATDome.html
+.. image:: https://img.shields.io/badge/GitHub-green.svg
+    :target: https://github.com/lsst-ts/ts_ATDome
+.. image:: https://img.shields.io/badge/Jira-green.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_ATDome
 
-.. _lsst.ts.ATDome-using:
+.. _Overview:
 
-Using lsst.ts.ATDome
-====================
+Overview
```

Comment:
That is an excellent suggestion. Thank you very much for the suggested text.

Reply:
I looked at https://obs-controls.lsst.io/System-Architecture/Control-Packages/index.html and did not find any mention of ATDome or ATDomeTrajectory so I'm inclined to leave that bit out. Maybe it needs updating, but I hope the User Guide will suffice. ATDome is simple.

Reply:
Yes, the content on that page is sorely lacking at the moment. I'm actually meeting with Tiago in ~20 minutes to talk about how we're going to flush that out. ATDome and ATDomeTrajectory are part of the ATCS class though (which is linked from that page, but I'll save you the hunt https://ts-observatory-control.lsst.io/py-api/lsst.ts.observatory.control.ATCS.html#lsst.ts.observatory.control.ATCS).
At this point in time i'd say it's better to "over-link" than "under-link" (if that makes sense), but i'm fine with whatever you choose.

Reply:
I am not yet comfortable linking to that page. I think it can be added if and when it looks more relevant. Links can so easily go stale that I think it's safer to link the other direction.